test/e2e/scripts/prow_run_smoke_test.sh

```
@@ -1,2 +1,200 @@
+  done
+
+  if [[ -e "$dir/$marker" ]]; then
+    printf '%s\n' "$dir"
+  else
+    echo "Error: couldn’t find '$marker' in any parent of '$start_dir'" >&2
+    return 1
+  fi
+}
+
+# Configuration
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] SCRIPT_DIR appears unused. Verify use (or export if used externally).

(SC2034)